app/views/vacancies/_vacancy.html.haml

```diff
@@ -1,23 +1,25 @@
-%li.vacancy
+%li.vacancy.bottom-border{ class: 'govuk-!-padding-0'}
```

Comment:
Can you amend the spacing so that it's `{ class: 'govuk-!-padding-0' }` instead? I think that should be more consistent with other existing haml files in the project